### ISSUE TITLE
Importing the multiarray numpy extension module failed. 

### ISSUE THREAD

**Issue by lilolalo** [numpy/numpy#17025]
Hello, I'm trying to run MontePython code, but am getting this error whenever I try to run using the PLC (Planck Likelihood Code) likelihoods (e.g. input/base.param below instead of the example.param). I have tried reinstalling numpy and Python. 

### Reproducing code example:
```
python montepython/MontePython.py run -o TestRun -p input/base.param
```

### Error message:
```
ImportError: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
Here is how to proceed:
- If you're working with a numpy git repository, try `git clean -xdf`
  (removes all files not under version control) and rebuild numpy.
- If you are simply trying to use the numpy version that you have installed:
  your installation is broken - please reinstall numpy.
- If you have already reinstalled and that did not fix the problem, then:
  1. Check that you are using the Python you expect (you're using /opt/local/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python),
     and that you have no directories in your PATH or PYTHONPATH that can
     interfere with the Python and numpy versions you're trying to use.
  2. If (1) looks fine, you can open a new issue at
     https://github.com/numpy/numpy/issues.  Please include details on:
     - how you installed Python
     - how you installed numpy
     - your operating system
     - whether or not you have multiple versions of Python installed
     - if you built from source, your compiler versions and ideally a build log

     Note: this error has many possible causes, so please don't comment on
     an existing issue about this - open a new one instead.

Original error was: dlopen(/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/numpy/core/_multiarray_umath.so, 2): Symbol not found: ___addtf3
  Referenced from: /opt/local/lib/libgcc/libquadmath.0.dylib
  Expected in: /usr/lib/libSystem.B.dylib
 in /opt/local/lib/libgcc/libquadmath.0.dylib

```

### Numpy/Python version information:

My numpy version is 1.16.6, and I installed it using MacPorts ```sudo port install py27-numpy```. My Python is 2.7.18 (I must use v2.7 for the code), installed with MacPorts also I believe ```sudo port install python27```.  The output of ```which python``` is ```/opt/local/bin/python```. I'm using Mac OS 10.15.4. I also have Python version 3.8.3 installed on my system.

I'm not sure if it has something to do with where my numpy is installed/how to change it exactly. I am also getting an error (possibly unrelated) which tells me ```No suitable python distribution found Cause : The configuration failed``` even though the correct version is installed. Any help would be greatly appreciated, I've been stuck with this issue for a few days...

**Comment by rossbar** [comment]
Please take a look at this [troubleshooting guide](https://numpy.org/devdocs/user/troubleshooting-importerror.html), paying particular attention to the steps related to PATH and PYTHONPATH.

**Comment by JFD-Mtl** [comment]
Hi, I am having a similar problem.
NumPy runs fine with the conda environment activated but not when running from a new windows shell.
Activating the conda environment manually is not an option as the python script is intended to be called from a VBA subroutine from an existing excel document. The script uses numpy, numpy-stl and pythonocc-core (from conda-forge).

This runs fine:
```
C:\>conda activate base
C:\>python
>>>import numpy
```

This does not:
```
C:\>conda deactivate
C:\>python
>>>import numpy
```

I currently have only one python interpreter installed and my environment variables are fine.
print(sys.executable) == "C:\anaconda3\python.exe"
print(os.environ.get('PYTHONPATH')) == "C:\anaconda3\"
print(os.environ.get('PATH')[:35]) == "C:\anaconda3\;C:\anaconda3\Scripts\;"
 
I had it running fine without a conda environment activated on a previous installation when installed with pip.
It does not seem to work when installed with conda.
I tried to uninstall numpy and reinstall it. pip refuses to install it on a conda environment.
I reinstalled anaconda, did not work. Reinstalled only numpy with conda, did not work.


```
Traceback (most recent call last):
  File "C:\anaconda3\lib\site-packages\numpy\core\__init__.py", line 24, in <module>
    from . import multiarray
  File "C:\anaconda3\lib\site-packages\numpy\core\multiarray.py", line 14, in <module>
    from . import overrides
  File "C:\anaconda3\lib\site-packages\numpy\core\overrides.py", line 7, in <module>
    from numpy.core._multiarray_umath import (
ImportError: DLL load failed while importing _multiarray_umath: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\anaconda3\lib\site-packages\numpy\__init__.py", line 142, in <module>
    from . import core
  File "C:\anaconda3\lib\site-packages\numpy\core\__init__.py", line 50, in <module>
    raise ImportError(msg)
...
Importing the numpy C-extensions failed.
...

  * The Python version is: Python3.8 from "C:\anaconda3\python.exe"
  * The NumPy version is: "1.18.5"
```

Thank you!

**Comment by mattip** [comment]
@JFD-Mtl This is a general issue with anaconda, not numpy. As you discovered, activating an anaconda environment is required to use conda. Perhaps other forums could help you solve this problem.

**Comment by JFD-Mtl** [comment]
Hi,
It is probably caused by conda but I will put the solution I found since I some people might end up here for help.
The issue was only occuring with Numpy and Numpy-stl, it is probably due to conda hiding it in subfolders.

Might be worth putting in the troubleshooting guide?

I had to include a bunch of other directories to my PATH variable.
C:\anaconda3
C:\anaconda3\Library\mingw-w64\bin
C:\anaconda3\Library\usr\bin
C:\anaconda3\Library\bin
C:\anaconda3\Scripts
C:\anaconda3\bin
C:\anaconda3\condabin

Source:
https://stackoverflow.com/questions/55015943/how-do-i-activate-python-anaconda-and-run-script-from-regular-command-prompt-in
